goss/README.md

```diff
@@ -0,0 +1,5 @@
+# Goss
+Goss is a standalone Go binary similar to servspec, which is used to validate server state: <https://github.com/aelsabbahy/goss>
+
```

Comment:
Perhaps it would be better if the `goss` binary is pulled into the image from the [release page](https://github.com/aelsabbahy/goss/releases)?

Reply:
Yea this is a better method that including it in the repo, I'll add download command to the setup script.